Doc/whatsnew/3.7.rst

```diff
@@ -768,17 +1490,68 @@ Add new clock identifiers:
   the time the system has been running and not suspended, providing accurate
   uptime measurement, both absolute and interval.
 
-Added functions :func:`time.thread_time` and :func:`time.thread_time_ns`
-to get per-thread CPU time measurements.
+The new :func:`time.thread_time` and :func:`time.thread_time_ns` functions
+can be used to get per-thread CPU time measurements.
 (Contributed by Antoine Pitrou in :issue:`32025`.)
 
+The new :func:`time.pthread_getcpuclockid` function returns the clock ID
+of the thread-specific CPU-time clock.
+
+
+timeit
+------
+
+:func:`timeit.timeit` autorange now uses a single loop iteration if
+the benchmark takes less than 10 seconds, instead of 10 iterations.
+``python3 -m timeit -s 'import time' 'time.sleep(1)'`` now takes 4 seconds
+instead of 40 seconds.
+(Contributed by Victor Stinner in :issue:`28240`.)
+
+
+tkinter
+-------
+
+The new :class:`tkinter.ttk.Spinbox` class is now available.
+(Contributed by Alan Moore in :issue:`32585`.)
+
+
+tracemalloc
+-----------
+
+:class:`tracemalloc.Traceback` behaves more like regular tracebacks,
+sorting the frames from oldest to most recent.
+:meth:`Traceback.format() <tracemalloc.Traceback.format>`
+now accepts negative *limit*, truncating the result to the
+``abs(limit)`` oldest frames.  To get the old behaviour, use
+the new *most_recent_first* argument to ``Traceback.format()``.
+(Contributed by Jesse Bakker. in :issue:`32121`.)
+
+
+types
+-----
+
+The new :class:`~types.WrapperDescriptorType`,
+:class:`~types.MethodWrapperType`, and :class:`~types.MethodDescriptorType`
+classes are now available.
+(Contributed by Manuel Krebber and Guido van Rossum in :issue:`29377`.)
+
+The new :class:`~types.ClassMethodDescriptorType` is used to describe
+class and static methods of builtin types.
+(Contributed by Serhiy Storchaka in :issue:`32265`.)
+
+The new :func:`types.resolve_bases` function resolves MRO entries
+dynamically as specified by :pep:`560`.
+(Contributed by Ivan Levkivskyi in :issue:`32717`.)
+
+
```

Comment:
There are some perf improvements in `typing` module due to PEP 560, I would probably mention 6-7x faster import of `typing`, but there are other perf improvements. They are all in the same PR https://github.com/python/cpython/pull/4906

Reply:
Perf improvements should be in the "Optimizations" section.